cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -83,7 +89,7 @@ def _provisioner(template, provisioning_data, delayed=None):
         fill(provisioning_form, provisioning_data, action=provisioning_form.submit_button)
         flash.assert_no_errors()
 
-        request.addfinalizer(lambda: cleanup_vm(vm_name, provider))
+        request.addfinalizer(lambda: remove_vm(vm_name, provider))
```

Comment:
cleanup_vm doesn't stop VM before removing it, and additionally it catches all exceptions and makes VM stay there silently when something goes wrong. We can't have that in our infrastructure.